tests/run/const_variable.pyx

```diff
@@ -0,0 +1,54 @@
+# mode: run
+
+cdef const int int_constant = 50
+cdef const int int_constant2 = int_constant
+cdef const int int_constant3 = 5
+cdef const int *pint_constant = &int_constant
+cdef const int int_sum_constant = 50 + 50
+# FIXME:
+# cdef const int int_sum_constant = 10 + int_constant
+
+cdef const float float_constant = 50.5
+# FIXME:
+# cdef const float float_sum_constant = 50.2 + 50.3
+# cdef const float float_sum_constant = 50.2 + float_constant
```

Comment:
@scoder @da-woods the following commented lines are failing now. Should I fix it in this PR (not sure how complex this can be) or separate PR should be created.

Reply:
I think this is potentially a quite difficult problem. It comes down to the "constant at compile time" to "constant at runtime" issue. Needs a little more thought (and also a bit of investigation about exactly how C handles it, because it'll affect the C code we need to generate)